src/views/Home/index.tsx

```diff
@@ -46,6 +45,19 @@ const Home = ({ history }: HomeProps) => {
               label="Go to Intake Request"
             />
           );
+        case 'SUBMITTED':
+          return (
+            <ActionBanner
+              key={intake.id}
+              title="Impact Analysis Network: Business Case"
```

Comment:
I need to check on this. I think "Impact Analysis Network" is placeholder text for the request name